src/utils/isPlainObject.js

```diff
@@ -1,6 +1,6 @@
 /**
- * @param {any} obj The object to inspect.
- * @returns {boolean} True if the argument appears to be a plain object.
+ * @param {*} obj The object to inspect.
```

Comment:
Hmm ok, I just taken it from http://usejsdoc.org/tags-param.html

![screen shot 2015-07-27 at 16 10 25](https://cloud.githubusercontent.com/assets/1110551/8908048/17d8d554-347a-11e5-8441-8e860bfa62e7.png)